packages/backend/storage/src/lib/__tests__/object_store.test.ts

```
@@ -0,0 +1,97 @@
+// packages/backend/storage/src/lib/__tests__/object_store.test.ts
+//
+// C-454 AC-1, AC-4: Unit tests for ObjectStore drivers.
+
+import { beforeEach, describe, expect, mock, test } from 'bun:test';
```

Biome (2.5.8)

[error] Several of these imports are unused.

(lint/correctness/noUnusedImports)